src/content/1.7/faq/product.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Product FAQ
+---
+
+# Product FAQ
+
+- [Product cover](#product-cover)
+
+## Product cover
+{{< minver v="1.7.7" title="true" >}}
+
+**Q:** How can I override the cover image of my products?
+
+**A:** By default, when a product is displayed in a list its cover image is used, it is configurable in the BackOffice and is set in product properties via the `cover_image_id` property.
```

Comment:
It's not really a property :thinking: 
`$product` is an array, not an object

Reply:
```suggestion
**A:** By default, when a product is displayed in a list its cover image is used, it is configurable in the BackOffice and is set in product properties via the `cover_image_id` key.
```

Reply:
I used the term `property` because it was in the context of the `getProductProperties` function, besides `$product` is an array at the beginning of the workflow and is then converted to a `ProductLazyArray` which is a mix between an array and an object 😅

But I see your point and I agree so to avoid any misunderstanding I used the term `key` instead as suggested